quizapp/src/main/java/billennium/quizapp/controller/QuizController.java

```diff
@@ -18,8 +20,8 @@
 
     private final QuizService quizService;
 
-    @GetMapping(ID_PARAM)
-    public QuizDefinitionDto getQuizFirstQuestion(@PathVariable String id) {
-        return quizService.getFirstQuizQuestion(id);
+    @PostMapping(ID_PARAM)
+    public QuizDefinitionDto playQuiz(@PathVariable String id, @RequestBody AnswersDto answersDto) {
```

Comment:
chodzi mi po głowie że nie powinno się mieszać w jednej metodzie zmiennych z URL i z requestBody ale nie mogę znaleźć żadnego uzasadnienia w necie.

Reply:
też poszukam póki co zrefaktorowałem logikę w serwisie i wyglada że śmiga. Mamy zapis i na nawet na froncie ładnie to wygląda